inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_gather.cpp

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "vpu/ngraph/transformations/dynamic_to_static_shape.hpp"
```

Comment:
Did you just introduce cyclic include?

Reply:
hm, I somehow did not pay attention to this, since all includes were in `cpp` files